edgebuilder-install.sh

```diff
@@ -560,6 +568,10 @@ while [ "$1" != "" ]; do
             INSTALL_DOCKER=true
             shift
             ;;
+        --no-frp)
+            NO_FRP=true
```

Comment:
In the server components pr we use the variable ENABLE_FRP, which is true for the path where we want to enable it. In the installer we use NO_FRP which is true for when we don't want to enable it. Can this be consistent (easiest to do it here probably)? Just name the variable ENABLE_FRP or INSTALL_FRP. Could help to make more sense in the future if someone ever works between the two files at once

Reply:
@jakeiotechsys the user-facing/interface varaible is no-frp and this is the style we follow in server like no-ssl and even though I also intially prefered it to be called ebale/install ( so positive ), Tom suggested that I should use no-frp as that is explicitly what the user wants, same like no-ssl and I agree because the use case is only triggered when the user dont want frp like when they dont want ssl. Inside the code, I use the +ve local varaibles just to be easy to work with, though no need

Reply:
Ok yes fair point about following the path of what the user wants as we've done with no_ssl, thanks for sharing that.